### ISSUE TITLE
Feature request: Add a backend support e.g. GPU on different machine.

### ISSUE THREAD

**Issue by idhamari** [lassoan/SlicerTotalSegmentator#47]
It would be nice to add a backend support so the extension runs using a GPU on a different machine e.g.:

1. The user installs the backend, e.g. flask server, and required libs
2. In 3D Slicer, a user adds the GPU server, the extension sends the input image and other parameters, the backend runs the segmentation task, and sends the result back to 3D Slicer.

**Comment by lassoan** [comment]
I agree that this can be useful and there is an example [flask server for TotalSegmentator v1](https://github.com/wasserth/TotalSegmentator/blob/v1.5.7/server.py), so it would not be that much work. However, from the example of MONAILabel we see that very few users can and are interested in setting up a server (even if it is just a single docker command).

Therefore, I would not invest time into this myself, but would be happy to merge a pull request that adds the frontend implementation to this extension and describes how to set up your own server.

**Comment by idhamari** [comment]
Thanks for the info and the link. I will try to add a PR if no one does. I suggest leaving this FR open as a reminder.
